ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ContextualServicesReporterOptions.java

```diff
@@ -30,6 +30,19 @@
 
   void setReportingEnabled(ValueProvider<Boolean> value);
 
+  @Description("Duration window for aggregation of reporting requests.")
+  @Default.String("10m")
+  String getAggregationWindowDuration();
+
+  void setAggregationWindowDuration(String value);
```

Comment:
@whd this is being added as a compile-time parameter.  Is there anything else that needs to be done?  We can use the default value for now

Reply:
If we need to set this in ops logic we can use https://github.com/mozilla-services/cloudops-infra/blob/master/projects/beam/Jenkinsfile.build#L136, but if we're using the default value I don't think anything else needs to change.

If we need to change this at some point without a corresponding code change it will require replacing the existing template since we don't currently have versioning on compile time parameters. That or we can just update the code default every time we want to change this.